src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FruitShopService {
+    void processOfOperations(List<FruitTransaction> fruitTransactions);
```

Comment:
The method name 'processOfOperations' is not very descriptive. Consider renaming it to something more meaningful, such as 'processTransactions'.